docs/source/models/gbd2017/causes/ischemic_heart_disease/index.rst

```diff
@@ -11,20 +11,58 @@ Ischemic heart disease (IHD) is a non-communicable cardiovascular disease which
 
 Acute coronary events may cause symptoms such as angina, cold sweats, dizziness, nausuea, neckpain, shortness of breath, sleep disturbances, or weakness. 
 
-Chronic ischemic heart disease can cause signs and symptoms such as angina, anxiety or nervousness, fatigue, or neckpace. 
+Chronic ischemic heart disease can cause signs and symptoms such as angina, anxiety or nervousness, fatigue, or neckpain. 
 
 .. _NIH: https://www.nhlbi.nih.gov/health-topics/ischemic-heart-disease
 
 
-Modeling Ischemic Heart Disease in GBD 2017
+Modeling Ischemic Heart Disease (IHD) in GBD 2017
 -------------------------------------------
 
 Myocardial infarction (MI) in GBD 2017
 ++++++++++++++++++++++++++++++++++++++
 
-
-Modeling Strategy for Ischemic Heart Disease
-++++++++++++++++++++++++++++++++++++++++++++
+GBD 2017 models IHD using Myocardial infarction (MI) sequelae to estimate the prevalence of IHD, due to the challenges of disease detection and varying symptoms across the population.
+
+
+1. Acute myocardial infarction (MI): Definite and possible MI according to the third universal
+definition of myocardial infarction:
+
+a. When there is clinical evidence of myocardial necrosis in a clinical setting consistent with
+myocardial ischemia or
+b. Detection of a rise and/or fall of cardiac biomarker values and with at least one of the following: 
+  i. symptoms of ischaemia, 
+  ii. new or presumed new ST-segment-T wave changes or new left bundle branch block, 
+  iii. development of pathological Q waves in the 277 ECG, 
+  iv. imaging evidence of new loss of viable myocardium or new regional wall motion abnormality, or 
+  v. identification of an intracoronary thrombus by angiography or autopsy.
+c. Sudden (abrupt) unexplained cardiac death, involving cardiac arrest or no evidence of a noncoronary cause of death
+d. Prevalent MI is considered to last from the onset of the event to 28 days after the event and is divided into an acute phase (0–2 days) and subacute (3–28 days).
+
+2. Chronic IHD
+a. Angina; clinically diagnosed stable exertional angina pectoris or definite angina pectoris
+according to the Rose Angina Questionnaire, physician diagnosis, or taking nitrate
+medication for the relief of chest pain.
+b. Asymptomatic ischemic heart disease following myocardial infarction; survival to 28 days
+following incident MI. The GBD study does not use estimations.
```

Comment:
Can you clarify the 'The GBD study does not use estimations' comment?

Reply:
My text cut off. Fixed it! Thanks for pointing this out.